css/components/avatar.css

```diff
@@ -1,74 +1,74 @@
 .avatar {
-	display: flex;
-	align-items: center;
-	justify-content: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
I think this could be extracted as a separate utility class, since, it is being used at many places here

Reply:
I have seprate utility classes in https://github.com/ttahm3d/chunks/blob/dev/css/components/layout.css. I don't want user to enter another class.